packaging/macos/uninstall.sh

```
@@ -210,80 +226,83 @@ if [[ "${PURGE}" == "true" \
+#
+# LEGACY_INSTALL_PREFIX is /Library/DefenseClaw/ — binary-only in the
+# old layout (runtime lived under LEGACY_SUPPORT_DIR), so it's safe to
+# remove wholesale in either mode.
+if [[ "${PURGE}" == "true" ]]; then
+  if [[ -d "${INSTALL_PREFIX}" ]]; then
+    log "removing ${INSTALL_PREFIX}"
+    rm -rf "${INSTALL_PREFIX}"
+  fi
+elif [[ -d "${INSTALL_PREFIX}/bin" ]]; then
+  log "removing ${INSTALL_PREFIX}/bin (runtime/config preserved for reinstall)"
+  rm -rf "${INSTALL_PREFIX}/bin"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /bin .

(SC2115)